package.json

```diff
@@ -41,7 +41,9 @@
     "typescript": "^3.8.2"
   },
   "dependencies": {
+    "@types/bn.js": "^4.11.6",
     "bech32": "^1.1.3",
+    "blake-hash": "^2.0.0",
```

Comment:
Did you add bn and blake-hash? If so you also need to commit `package-lock.json` so that we can check the actual size increase.

Reply:
@makoto Done. Review